src/status_im2/contexts/chat/messages/navigation/view.cljs

```diff
@@ -60,13 +61,13 @@
         :blur-type   (colors/theme-colors :light :dark)
         :blur-radius (if platform/ios? 20 10)
         :style       {:flex 1}}]]
-
      [rn/view
       [rn/view {:style style/header-container}
```

Comment:
Redundant view.
Will remove once PR is reviewed, otherwise change/formatting messes up the file and hides actual change.